### PR TITLE
Add user stories

### LINKED ISSUE #32: Create a list of user stories
Proposed list of our user stories for the minimal viable project: https://hackpad.com/Wizlist-User-Stories-AQauQ1ljg2b

### PR DESCRIPTION
**Pull request by student-tomasz**
Closes #32

### CODE REVIEW

**Comment by shadre** [comment]
:+1:

**Comment by kamildebicki** [comment]
:+1:

File: doc/user_stories.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Wizlist — User Stories
+
+### Guest:
+
+- _As a_ Guest _I want to_ access a wish list from an invitation link _so that_ I can reserve one or more gifts.
+
+### Not logged-in Organizator:
+
+- _As a_ Not logged-in Organizator _I want to_ sign up _so that_ I can create my own wish lists.
```
Comment (agalkiewicz) [inline doc/user_stories.md]: I think "organizer" is the correct form.

**Comment by gotar** [comment]
👍 ok you can merge

**Pull request merged**